xsl/templates.xsl

```diff
@@ -186,7 +186,17 @@ SOFTWARE.
   </xsl:template>
   <xsl:template name="par">
     <xsl:param name="text"/>
-    <xsl:value-of select="$text"/>
+    <xsl:analyze-string select="$text" regex="(§[0-9]*)">
```

Comment:
@paulodamaso `(§[0-9]*)` -> `(§[0-9]+)`, we expect a paragraph to contain at least one number, not zero or more

Reply:
@paulodamaso also as you need only the number part, you should move the parentheses after the paragraph character: `§([0-9]+)`